### DISCUSSION TITLE
The next beta before solstice

### DISCUSSION THREAD

**Discussion by ikedas** [sympa-community/sympa#1610] [Release works]
Hi @everyone, we'd be better to release new beta in this month, around 23 Nov.

**Comment by racke** [comment]
Good idea :+1:

**Comment by ikedas** [comment]
...and beta 2 would be out in this week, Tuesday or Wednesday.

**Comment by racke** [comment]
Good :+1:

**Comment by ikedas** [comment]
Done!

**Comment by ikedas** [comment]
At least sympa-community/sympa#820 will be fixed, the last beta should be released, maybe around this weekend... 15th?

**Comment by racke** [comment]
:+1: - so no changes after that date save translations and show stoppers?

**Comment by ikedas** [comment]
Yes.  As [release note](https://github.com/sympa-community/sympa/releases/tag/6.2.49b.2) says, "_Translations added on translation site before 19th December 00:00 UTC will be shipped with the stable release._"

**Comment by ikedas** [comment]
Done!

**Comment by racke** [comment]
Thanks!

**Comment by ikedas** [comment]
Patch release will come to fix sympa-community/sympa#831.

**Comment by racke** [comment]
:+1:

**Comment by ikedas** [comment]
6.2.52 will be released in 27 Dec. Draft [announcement](https://github.com/sympa-community/sympa/releases) (sign-in needed).

**Comment by racke** [comment]
I did a few minor updates to hopefully improve the language :-).

**Comment by ikedas** [comment]
Of course it's also included!
